symplyphysics/definitions/centripetal_acceleration_is_squared_velocity_by_radius.py

```diff
@@ -0,0 +1,31 @@
+from symplyphysics import (
+    symbols, Eq, pretty, Quantity, units, solve,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+## Centripetal acceleration is type of acceleration which is perpendicular to velocity vector and directed to the curve center.
+## Centripetal acceleration only changes the velocity vector direction and not the velocity vector length.
+## Definition: an = V**2 / R, where
```

Comment:
AFAIK this law can be derived from rotating coordinate system. So it should be in laws, not in definitions.